### DISCUSSION TITLE
Opcodes. Why?

### DISCUSSION THREAD

**Discussion by danieloneill** [Palakis/obs-websocket#943] [General]
I don't understand the advantage of sending ```op: 0``` instead of ```op: "hello"```.

The operation name maps to the number, which is documented in protocol.md. Then in the code those numbers map to enumerated names of those things, which the code internally uses. Why use a numeric opcode instead of just the name of the actual operation?

**Comment by tt2468** [comment]
For a very slight bandwidth overhead improvement. Also because the v5 API is inspired by Discord's gateway API and that's what they do.
